superagi/tools/google_calendar/event_details_calendar.py

```diff
@@ -0,0 +1,43 @@
+import base64
+from typing import Any, Type
+from pydantic import BaseModel, Field
+from superagi.tools.base_tool import BaseTool
+from superagi.helper.google_calendar_creds import GoogleCalendarCreds
+
+class EventDetailsCalendarInput(BaseModel):
+    event_id: str = Field(..., description="The id of event to be fetched from Google Calendar. if no value is given keep default value is None")
+
+class EventDetailsCalendarTool(BaseTool):
+    name: str = "Fetch Google Calendar Event"
+    args_schema: Type[BaseModel] = EventDetailsCalendarInput
+    description: str = "Fetch an event from Google Calendar"
+
+    def _execute(self, event_id: str):
+        toolkit_id = self.tool_kit_config.tool_kit_id
+        service = GoogleCalendarCreds().get_credentials(toolkit_id)
+        if service["success"]:
+            service = service["service"]
+        else:
+            return f"Kindly connect to Google Calendar"
+        if event_id == "None":
+            return f"Add Event ID to fetch details of an event from Google Calendar"
+        else:
+            decoded_id = base64.b64decode(event_id)
+            eid = decoded_id.decode("utf-8")
+            eid  = eid.split(" ", 1)[0]
```

Comment:
can be done in single line.
base64.b64decode(event_id).decode("utf-8").split(" ", 1)[0]